cmd/loop/loopout.go

```diff
@@ -115,6 +121,10 @@ func loopOut(ctx *cli.Context) error {
 	}
 
 	limits := getLimits(amt, quote)
+	// If configured, use the specified maximum swap routing fee.
+	if ctx.IsSet("max_swap_routing_fee") {
+		*limits.maxSwapRoutingFee = btcutil.Amount(ctx.Int64("max_swap_routing_fee"))
```

Comment:
Max line len of 80 chars exceeded. We use tab size 8.

Reply:
@joostjager, do we have any tool for code formatting?

Reply:
There is the `make fmt` command that makes sure that the code is formatted in the way the linter expects.
But long lines aren't automatically wrapped. See [the contribution guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#48-additional-style-constraints-on-top-of-gofmt) to see how we wrap method calls.

This example would look like this:

```
 		*limits.maxSwapRoutingFee = btcutil.Amount(
 		 		ctx.Int64("max_swap_routing_fee"),
 		) 
```

Reply:
Thanks @guggero.

CR feedbacks have been addressed, PTAL, thx!

cc @joostjager